### PR TITLE
Added Old North Arabian language pack

### PR DESCRIPTION
**Pull request by wahballat**
Please review my pull request, please let me know if I need to fix anything. 

I had issues compiling as UTF-8 on Windows, and had to comment out building the dictionary. The changes I made to:
addons/gradle/language_pack_lib.gradle
buildSrc/build.gradle 
buildSrc/src/main/java/Parser.java

should probably not be committed, but were necessary for me to build.

### CODE REVIEW

File: addons/gradle/language_pack_lib.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.library'
 
 apply from: "$rootDir/gradle/android_general.gradle"
 
-apply plugin: 'make-dictionary'
+//apply plugin: 'make-dictionary'
```
Comment (menny) [inline addons/gradle/language_pack_lib.gradle:5]: revert this

File: addons/languages/oldnortharabian/apk/build.gradle [hunk 2]
```diff
@@ -0,0 +1,13 @@
+ext.shouldBePublished = true
```
Comment (menny) [inline addons/languages/oldnortharabian/apk/build.gradle:1]: Since this is a new pack (that has not been published yet), you need to set it `false`. So:
```
ext.shouldBePublished = false
ext.notPublishingReason = "Initial version has not been published yet"
```

File: buildSrc/build.gradle [hunk 3]
```diff
@@ -18,10 +18,6 @@ dependencies {
 
 gradlePlugin {
     plugins {
-        makeDictionaryPlugin {
```
Comment (menny) [inline buildSrc/build.gradle:21]: revert this change

File: buildSrc/src/main/java/Parser.java [hunk 4]
```diff
@@ -166,11 +166,11 @@ private void addWordsFromInputStream(final long inputSize, InputStreamReader inp
 
     private char fixup(int intChar) {
         switch (intChar) {
-            case '’':
-                return '\'';
-            case '”':
-            case '“':
-                return '\"';
+//            case '’':
```
Comment (menny) [inline buildSrc/src/main/java/Parser.java:169]: revert this change

**Review by menny — CHANGES_REQUESTED** [review]
thanks.
Please revert the changes you made to the `gradle` scripts. Also, Since this is a new pack, you must mark it as not-to-publish. I added a comment to how to do so.

**Comment by wahballat** [comment]
I made the changes as suggested.

How can this language pack be published to the store? who can help with that?

Thank you

**Comment by wahballat** [comment]
How can I resolve the unsuccessful checks? Is the lack of a dictionary an issue? After submitting my pull request I noticed in some other language packs they use a dummy dictionary, sometimes copied from another language pack.

**Comment by menny** [comment]
> How can I resolve the unsuccessful checks? Is the lack of a dictionary an issue? After submitting my pull request I noticed in some other language packs they use a dummy dictionary, sometimes copied from another language pack.

I pushed a commit into your branch to disable dictionary generation for now. (please pull your branch to get that commit)

**Review by menny — APPROVED** [review]

**Comment by menny** [comment]
thank you!
